content/packages/ra-mirrors.txt

```diff
@@ -11,3 +11,4 @@ http://mirrors.ustc.edu.cn/anthon/openra/ra-packages.zip
 http://mirror.oss.maxcdn.com/anthonos/openra/ra-packages.zip
 http://81.169.246.181/ra-packages.zip
 http://zoners.eu/openra/ra-packages.zip
+http://178.63.32.174/ra-music.zip
```

Comment:
Oh, that was a small copy and paste problem with a huge impact. This probably requires automated checks as it stayed undetected for such a long time.